head_ref/src/HeadReference.cpp

```diff
@@ -342,7 +371,7 @@ bool HeadReference::targetToPanTilt(const tf::Stamped<tf::Point>& target, double
 bool HeadReference::getJointsInfo()
 {
   urdf::Model model;
-  if (!model.initParam(tf_prefix_ + "/robot_description"))
+  if (!model.initParam("robot_description"))
```

Comment:
Question: Just to confirm. This is still able to read parameter "hero/robot_description" by namespacing the head ref node?

Reply:
Yes